rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -11,8 +11,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,5)" />
+        <sourceTechnology id="eap" versionRange="[6,7)" />
         <targetTechnology id="hibernate5" versionRange="[5,)" />
         <targetTechnology id="eap7" versionRange="[7,)" />
+        <targetTechnology id="eap" versionRange="[7,)" />
```

Comment:
Not that I care much :-), but this can possibly affect people migrating from XXX to EAP 6, no?